helix-core/src/test/java/org/apache/helix/integration/TestPartitionMovementThrottle.java

```diff
@@ -215,7 +215,7 @@ public void testANYtypeThrottle() throws InterruptedException {
     DelayedTransition.setDelay(20);
     DelayedTransition.enableThrottleRecord();
 
-    // start another 3 nodes
+    // start another 3 nodes 
```

Comment:
Please remove the extra space before merge.

Reply:
removed.